go/common/crypto/signature/signature.go

```diff
@@ -430,6 +437,35 @@ func (s *Signed) ToProto() *common.Signed {
 	}
 }
 
+// PrettySigned is used for pretty-printing signed messages so that
+// the actual content is displayed instead of the binary blob.
+//
+// It should only be used for pretty printing.
+type PrettySigned struct {
+	Body      interface{} `json:"untrusted_raw_value"`
+	Signature Signature   `json:"signature"`
+}
+
+// PrettyPrint returns a pretty-printed representation of the structure.
+func (p PrettySigned) PrettyPrint(prefix string, v interface{}) string {
+	pp := v.(*PrettySigned)
```

Comment:
Pretty sure that i'm missing something, but wouldn't the same be achieved with:
```
func (pp *PrettySigned) PrettyPrint(prefix string) string {
```

Reply:
Yeah it is not, but I think this can be changed, I was experimenting with the interface but now I see that it can be cleaned up, thanks!